managing-os/templates/fedora/upgrade-26-to-27.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: doc
+title: Upgrading the Fedora 26 Template to Fedora 27
+permalink: /doc/template/fedora/upgrade-26-to-27/
+redirect_from:
+- /doc/fedora-template-upgrade-26/
+- /en/doc/fedora-template-upgrade-26/
+- /doc/FedoraTemplateUpgrade26/
+- /wiki/FedoraTemplateUpgrade26/
+---
+
+Upgrading the Fedora 26 Template to Fedora 27
+=============================================
+
+This page provides instructions for performing an in-place upgrade of an
+installed Fedora 26 [TemplateVM] to Fedora 27. If you wish to install a new,
+unmodified Fedora 27 template instead of upgrading a template that is already
+installed in your system, please see the [Fedora TemplateVM] page instead.
+
+These instructions can also be used to upgrade a Fedora 25 TemplateVM to
+Fedora 27. Simply start by cloning `fedora-25` instead of `fedora-26` in the
+instructions below.
```

Comment:
For now the packages for fc27 are available only in current-testing repository. Should we add a note about that? Or delay announcing this page until migrated to current? Alternatively, ask users to test instruction and in that message add note about `--enablerepo=qubes-vm*current-testing`.

Reply:
I'll add a notice at the top of the page stating this.

But what about 3.2? Are the packages available in `current-testing` for 3.2 yet?

Reply:
Not yet.

Reply:
Ok, I'll make the note specify that they're only available for 4.0.